guides/key-performance-indicators-with-sigma.md

```diff
@@ -0,0 +1,483 @@
+	b. Due to the differences in date mapping between the `.csv` and the `DATE` data-type when creating the table, we will temporarily set the **datestyle** to `euro`.
+
+	```sql
+	set datestyle = euro;
+	show datestyle;
+	 DateStyle 
+	-----------
+	 ISO, DMY
+	```
+	c. The below code is used to create the sales_data table. Note: we will create a [columnar](https://docs.hydra.so/organize/data-modeling/row-vs-column-tables) table, which makes it much quicker to obtain all data for a particular column.
+
+	```sql
+	CREATE TABLE sales_data (
+
+	retailer_code INT,
+
+	product_number INT,
+
+	order_method_code INT,
+
+	date DATE,
+
+	quantity INT,
+
+	unit_price FLOAT,
+
+	unit_sale_price FLOAT
+
+	) USING columnar;
+	```
+
+	  d. Once the table has been created, upload the .csv by utilizing the following command:
+	  `\COPY sales_data FROM 'notebooks/data/ibm_gosales/go_daily_sales.csv' DELIMITER ',' CSV HEADER;`
+
+  
+
+  
+
+3. You should now have a sales-oriented table populated with approximately 150,000 rows.
+
+  
+
+  
+
+  
+
+  
+
+### Connecting Sigma with Hydra
+
+  
+
+  
+
+  
+
+  
+
+Now we can configure Sigma to access our sample data in Hydra.
+
+  
+
+  
+
+  
+
+  
+
+1. As mentioned earlier, create a [trial account](https://www.sigmacomputing.com/free-trial).
+
+  
+
+  
+
+  
+
+2. Open your Admin Portal by selecting **Administration** in the user menu at the top right of your screen.
+
+  
+
+  
+
+  
+
+3. Select the **Connections** page from the top left-hand panel.
+
+  
+
+  
+
+  
+
+4. Click the **Create Connection** button on the top right.
+
+  
+
+  
+
+  
+
+5. Choose a name for the connection (we went with **hydra-sales_data_2023**) and select PostgreSQL as the type of database you'd like to connect to.
+
+  
+
+![](/.gitbook/assets/.sigma-images/connection_details.png)
+
+  
+  
+
+  
+
+  
+
+  
+
+6. Retrieve your database credentials from your Hydra dashboard. You will need Hostname, User, Password, and Database. Keep all other options unchanged. Once all the information has been entered, select **Create** on the top right.
+
+  
+
+![](/.gitbook/assets/.sigma-images/connection-credentials.png)
+
+  
+
+  
+
+  
+
+  
+
+7. Once created, you will be able to view the connection details.
+
+  
+
+![](/.gitbook/assets/.sigma-images/connection-summary.png)
+
+  
+  
+  
+
+  
+
+### Creating Your First Hydra-backend Dashboard
+
+  
+
+  
+
+  
+
+1. Back on the main page, click **Create New** on the top left followed by selecting **Workbook**. You will see a banner entitled **DATA ELEMENTS** on the toolbar to the left.
+
+  
+
+  
+
+  
+
+2. Within the aforementioned toolbar, select **VIZ** followed by clicking on **TABLES AND DATASETS**. This will bring you to the Hydra database and will show you the available schemas. Your data should exist within the **public** schema.
+
+  
+
+![](/.gitbook/assets/.sigma-images/select-table.png)
```

Comment:
```suggestion
  ![](/.gitbook/assets/.sigma-images/select-table.png)
```

I think indenting the images will put the images inside each `li` element of the ordered list here (applies to each image in this section and the previous section) which should make the steps a little easier to follow.

Reply:
Done. All images referencing an `li` element should now be inside their respective element.